app/controllers/application_controller.rb

```diff
@@ -101,12 +102,21 @@ def append_info_to_payload(payload)
 
   def info_for_paper_trail
     {
-      user_id: current_user&.id,
+      user_id: true_user&.id,
```

Comment:
We had to replace `current_user` with `warden&.user` for the Rails migration... Any idea if `true_user` is going to work?

Reply:
Good question.  I'm not sure.  https://github.com/ankane/pretender is ready for rails 5.  I don't remember the "why" of the required change.  We'll have to test.

Reply:
Tested.  `true_user` does not work correctly in rails 5 either.  `warden&.user&.id` works in both scenarios.  Changing.